app/Main.hs

```diff
@@ -1,7 +0,0 @@
-module Main where
```

Comment:
I'd prefer not to chop this file out entirely. I agree it should disappear in the long term, but it's still useful for inspecting core.